cpp_linter/__init__.py

```diff
@@ -89,6 +90,9 @@ def main():
         tidy_review=is_pr_event and args.tidy_review,
         format_review=is_pr_event and args.format_review,
         num_workers=args.jobs,
+        extensions=args.extensions,
+        tidy_ignore=args.ignore_tidy,
+        format_ignore=args.ignore_format,
```

Comment:
Most of these args are just passing directly from the `args` namespace created from `parse_args()`. It might be more efficient to pass either <sup>1</sup>a derivative of the CLI args namespace that is specifically applied to tool analysis or <sup>2</sup>the entire `args` object.